dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -2472,7 +2511,22 @@ public static void HydraTask_cycleStoreVms() {
       int numToKill = TestConfig.tab().intAt(SnappyPrms.numVMsToStop, 1);
       int stopStartVms = (int) SnappyBB.getBB().getSharedCounters().incrementAndRead(SnappyBB.stopStartVms);
       Long lastCycledTimeForStoreFromBB = (Long) SnappyBB.getBB().getSharedMap().get(LASTCYCLEDTIME);
-      snappyTest.cycleVM(numToKill, stopStartVms, "storeVmCycled", lastCycledTimeForStoreFromBB, lastCycledTime, false);
+      snappyTest.cycleVM(numToKill, stopStartVms, "storeVmCycled", lastCycledTimeForStoreFromBB,
+          lastCycledTime, "server", false, false);
+    }
+  }
+
+  /**
+   * Concurrently stops a List of snappy locator VMs, then restarts them.  Waits for the
+   * restart to complete before returning.
+   */
+  public static void HydraTask_cycleLocatorVms() {
+    if (cycleVms) {
+      int numToKill = TestConfig.tab().intAt(SnappyPrms.numVMsToStop, 1);
+      int stopStartVms = (int) SnappyBB.getBB().getSharedCounters().incrementAndRead(SnappyBB.stopStartVms);
+      Long lastCycledTimeForStoreFromBB = (Long) SnappyBB.getBB().getSharedMap().get(LASTCYCLEDTIME);
+      snappyTest.cycleVM(numToKill, stopStartVms, "locatorVmCycled", lastCycledTimeForStoreFromBB,
```

Comment:
lastCycledTimeForStoreFromBB should be lastCycledTimeForLocatorFromBB, this can issues in future when we have missed HA tests